doi_to_org.py

```diff
@@ -305,7 +329,10 @@ def orgmode_from_bibentry(self, attached_file_name=None, attached_file_hash=None
         ))
 
     def generate_attachment_file_name(self):
-        return self.bibentry.title.replace(' ', '_') + self.attachment.extension
+        if self.pdfpath:
+            return self.bibentry.key + self.attachment.extension
+        else:
+            return self.bibentry.title.replace(' ', '_') + self.attachment.extension
```

Comment:
I don't get this change. This filename will be the name of the attached file, it is not necessary the same as the original file. So why do you want to use another naming convention depending on the origin of the file?

Reply:
And because of this change, `OrgEntry` has to know `pdfpath`. I don't think this should be the case, we just need to give to the `OrgEntry` an `Attachment`.

Reply:
Thanks for your feedback!

I added this in order to keep the previous filenames I had but I agree this was not the best solution.

Do you think adding an option to the `.doirc` file to keep names or select different strategies for naming would be a better way to achieve this?

Reply:
Yes, we could add an option to the `.doirc` file.
An easy way would be a boolean variable that tells if we keep the original name or not. A more complex solution could be a "format string" that tells what to use, like this:
```yaml
pdfname : "{bibtexkey}_{authorname}_{title}"
```
or :
```yaml
pdfname : "{filename}"
```
I am not sure about the best syntax however, and it will be harder to do.